packages/@uppy/utils/src/AbortController.js

```diff
@@ -1,6 +1,6 @@
 /**
  * Little AbortController proxy module so we can swap out the implementation easily later.
  */
-exports.AbortController = AbortController
-exports.AbortSignal = AbortSignal
+exports.AbortController = typeof AbortController !== 'undefined' ? AbortController : undefined
+exports.AbortSignal = typeof AbortSignal !== 'undefined' ? AbortSignal : undefined
```

Comment:
I think this would produce the same result while being less verbose.
```suggestion
exports.AbortController = globalThis.AbortController
exports.AbortSignal = globalThis.AbortSignal
```

Reply:
this won't work as globalThis is not defined on SSR (Node.js) environments, the fix I proposed works and is the way all SSR frameworks check for Browser only features.

Reply:
`globalThis` is available globally on all recent Node.js versions (≥ 12.0.0), are you sure it's not available in your SSR env?

Reply:
you are right, I just read about it, I thought it was a +16 thing